src/components/ProductMilestoneDeliveredArtifactsPage/ProductMilestoneDeliveredArtifactsPage.tsx

```diff
@@ -26,5 +29,19 @@ export const ProductMilestoneDeliveredArtifactsPage = ({ componentId = 'd1' }: I
     { componentId }
   );
 
-  return <ArtifactsList {...{ serviceContainerArtifacts, componentId }} />;
+  return (
+    <>
+      <Toolbar>
+        <ToolbarItem reservedWidth>
+          <TextContent>
+            <Text component="h2">Delivered Artifacts</Text>
+            <Text>
+              This list contains Artifacts delivered in the current Milestone. Each Artifact is represented by a PNC Identifier.
```

Comment:
`in the Milestone`, the "current milestone" collides with the current milestone of the version